nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -24,13 +24,97 @@
   </section>
   <section xml:id="backward-incompatibilities">
     <title>Backward Incompatibilities</title>
-    <itemizedlist spacing="compact">
+    <itemizedlist>
```

Comment:
Note that this file is generated and the change here is due to regenerating the files. Apparently it changes the `spacing` once there is a second (or more?) items in a list.